### PR TITLE
feat(universal): make wrapper work with universal deploy

### PR DESCRIPTION
**Pull request by tripodsan**

### CODE REVIEW

**Comment by tripodsan** [comment]
I'll also convert it to make the main function work with universal deploy

File: package.json [hunk 1]
```diff
@@ -31,7 +31,8 @@
   "homepage": "https://github.com/adobe/helix-status#readme",
   "dependencies": {
     "@adobe/helix-fetch": "1.9.2",
-    "@adobe/helix-log": "4.5.3"
+    "@adobe/helix-log": "4.5.3",
+    "node-fetch": "2.6.1"
```
Comment (trieloff) [inline package.json:35]: Can't we just use the imports from helix-fetch?

File: src/index.js [hunk 2]
```diff
@@ -14,6 +14,7 @@
 const fs = require('fs');
 const { error } = require('@adobe/helix-log');
 const fetchAPI = require('@adobe/helix-fetch');
+const { Response } = require('node-fetch');
```
Comment (trieloff) [inline src/index.js:17]:
```suggestion
const { Response } = require('@adobe/helix-fetch');
```

**Review by trieloff — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (tripodsan) [reply package.json:35]: I thought it's not ready yet?

Comment (tripodsan) [reply package.json:35]: a quick test yields:

```
error: test failed: 500 Cannot read property 'entries' of undefined
```

Comment (trieloff) [reply package.json:35]: @stefan-guggisberg it seems `@adobe/helix-fetch/FetchContext` does not export `Request` and `Response` – is there a reason not to export them?

**Review by trieloff — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (stefan-guggisberg) [reply package.json:36]: the reason were:
* they were not needed for the targeted use case (i.e. exposing `fetch(url, options)`)
* exporting the `Request` and `Response` classes from the underlying `fetch-h2` implementation would expose implementation specifics of `fetch-h2`; headers e.g. are guarded and only allow certain headers.
* exporting `Request` would require support for `fetch(request, options)`

for now I would stick the `node-fetch` `Request` and `Response` classes. v2 of `helix-fetch` will export general purpose `Request` and `Response` implementations.

**Pull request merged**